.github/actions/next-integration-stat/src/index.ts

```diff
@@ -566,13 +654,13 @@ function getTestSummary(
 
   if (fixedTests.length > 0) {
     ret += `\n:white_check_mark: **Fixed tests:**\n\n${fixedTests
-      .map((t) => `\t- ${t}`)
+      .map((t) => (t.length > 5 ? `\t- ${t}` : t))
```

Comment:
Maybe a comment explaining why 5 is significant?

Reply:
it's actually no significant random number 😅 I just picked small enough to ensure it have corresponding filename in the logs.